dashboard/app/bisect_test.go

```diff
@@ -1151,6 +1089,47 @@ func TestBugBisectionStatus(t *testing.T) {
 	c.client2.pollEmailBug()
 
 	// Receive the JobBisectCause and send cause information.
+	addBisectCauseJob(c, build)
+	// Advance time by 30 days and read out any notification emails.
+	{
+		c.advanceTime(30 * 24 * time.Hour)
+		msg := c.client2.pollEmailBug()
+		c.expectTrue(strings.Contains(msg.Body, "syzbot has bisected this issue to:"))
+		msg = c.client2.pollEmailBug()
+		c.expectTrue(strings.Contains(msg.Body, "Sending this report to the next reporting stage."))
+		msg = c.client2.pollEmailBug()
+		c.expectTrue(strings.Contains(msg.Body, "syzbot found the following issue"))
+	}
+
+	// Ensure expected results show up on web UI
+	bug, bugKey := getBug(c)
+	job, jobKey := getJob(c)
+	bugURL := fmt.Sprintf("/bug?id=%v", bugKey.StringID())
+	content, err := c.GET(bugURL)
+	c.expectEQ(err, nil)
+	c.expectEQ(bug.BisectCause, BisectYes)
+	c.expectTrue(bytes.Contains(content, []byte("Cause bisection: introduced by")))
+	c.expectTrue(bytes.Contains(content, []byte("kernel: add a bug")))
+	c.expectEQ(job.IsInvalid, false)
+
+	// Mark bisection as invalid.
+	_, err = c.AuthGET(AccessAdmin, "/admin?action=invalidate_bisection&key="+jobKey.Encode())
+	httpErr, ok := err.(HTTPError)
+	c.expectTrue(ok)
+	c.expectEQ(httpErr.Code, http.StatusFound)
+
+	// The invalidated bisection should have vanished from the web UI
```

Comment:
Do you also check whether syzbot re-creates the job after some cooldown?

Reply:
Could you please point to the line?
I just cannot find it without digging too deep into the test code.

Reply:
Just pushed the change here: https://github.com/google/syzkaller/pull/3874/files#diff-567d6d1d98ecb970e31b8456dd220abb69fc6d154e2480733965bc3e20103658R1123-R1129